docs/HowToUseTaskMonitor.md

```diff
@@ -0,0 +1,78 @@
+# How to use Task Monitor
+
+Task Monitor is a kernel thread which monitors the tasks/pthreads.
+It checks if registered tasks/pthreads are alive.
+
+If any task/pthread does not update its active status within registered interval, then Task Monitor will reset the board.
+
+## Contents
+- [Enable Task Monitor](#enable-task-monitor)
+- [Use Task Monitor](#use-task-monitor)
+
+
+## Enable Task Monitor
+To enable Task Monitor, please find four steps as shown below:
```

Comment:
Please find four steps to enable Task monitor as shown below:

Reply:
modified